cpmpy/transformations/normalize.py

```diff
@@ -34,3 +35,70 @@ def unravel(lst, append):
     unravel((cpm_expr,), append)
 
     return newlist
+
+
+def simplify_bool_const(lst_of_expr, num_context=False):
+    """
+    removes boolean constants from all CPMpy expressions
+    only resulting boolean constant is literal 'false'
+    - list_of_expr: list of CPMpy expressions
+    """
+
+    def _is_false(expr):
```

Comment:
I just merged Wout's _is_false_cst in utils, which also checks for numpy bool and should be used here instead of defining new ones

Reply:
Aha I missed that one, will change it